### DISCUSSION TITLE
Using i18n in helper/utils

### DISCUSSION THREAD

**Discussion by kelbertpaul** [nuxt-modules/i18n#1420] [Q&A]
Hi everyone,

I'm trying to use i18n inside a helper : 
![image](https://user-images.githubusercontent.com/55586343/159329338-1b887fe6-9892-4771-bd3b-f31e7e939675.png)

but I got this error : 
`Cannot read properties of undefined (reading '$t').`

I did try to import the plugin to refer to it but it doesn't seem to work neither ...

Here is the architecture of my project in case I did something wrong.

![image](https://user-images.githubusercontent.com/55586343/159328698-ff5fddac-c126-4d66-853e-9bbcb16a5087.png)

Thanks for you help.

**Comment by rchl** [comment]
It's available in certain contexts like Nuxt Context, Vue Instance and store. It can't be just freely imported from anywhere. You'd have to pass it to your utilities one way or another.
